Graphviz/DOT.sublime-syntax

```diff
@@ -1,62 +1,373 @@
+
+  literal-strings:
+    - include: special-strings
+    - include: literal-double-quoted-strings
+
+  special-strings:
+    - match: |-
+        (?x:
+          (\")
+          \b(
+            {{clustermode_attribute_types}}
+          | {{dirtype_attribute_types}}
+          | {{outputmode_attribute_types}}
+          | {{packmode_attribute_types}}
+          | {{pagedir_attribute_types}}
+          | {{quadtype_attribute_types}}
+          | {{ranktype_attribute_types}}
+          | {{rankdir_attribute_types}}
+          | {{smoothtype_attributes_types}}
+          | {{style_attribute_types}}
+          )\b
+          (\")
+        )
+      scope: >-
+        meta.string.dot
+        string.quoted.double.dot
+      captures:
+        1: punctuation.definition.string.begin.dot
+        2: support.constant.dot
+        3: punctuation.definition.string.end.dot
+
+  literal-double-quoted-strings:
+    - match: \"
       scope: punctuation.definition.string.begin.dot
-      push:
-        - meta_scope: string.quoted.double.dot
-        - match: '"'
-          scope: punctuation.definition.string.end.dot
-          pop: true
-        - match: \\.
-          scope: constant.character.escape.dot
-        - match: (\\)$\n?
-          captures:
-            1: punctuation.separator.continuation.line.dot
-        - match: '[{}|]'
-          scope: punctuation.separator.memory-block.dot
-    - include: braces
-    - include: brackets
-    - include: embedded-html
-    - include: comments
+      push: literal-double-quoted-string-body
+
+  literal-double-quoted-string-body:
+    - meta_include_prototype: false
+    - meta_scope: >-
+        meta.string.dot
+        string.quoted.double.dot
 
+    - match: \"
+      scope: punctuation.definition.string.end.dot
+      pop: 1
+    - match: '(\\){{eol}}'
+      captures:
+        1: punctuation.separator.continuation.line.dot
+
+    - include: illegal-newline
+    - include: literal-double-quoted-string-escapes
+
+    # TODO: https://graphviz.org/doc/info/shapes.html#record
+    - match: \{
+      scope: punctuation.section.block.begin.memory.dot
+    - match: \}
+      scope: punctuation.section.block.end.memory.dot
+    - match: '|'
+      scope: punctuation.separator.memory.dot
+
+  literal-double-quoted-string-escapes:
+    - match: \\\"
+      scope: constant.character.escape.dot
+
+    - include: literal-string-escapes
+
+  literal-string-escapes:
+    # https://graphviz.org/docs/attr-types/escString/
+
+    # \\ = backslash
+    # \n, \l, \r = alignment (center, left, right)
+    # \N, \H, \T = node name, head node name, tail node name
+    # \G = graph name
+    # \E = edge name
+    # \L = object's label attribute
+    - match: '\\[\\lnrEGHLNT]'
+      scope: constant.character.escape.dot
+
+    - match: \\.
+      scope: invalid.illegal.escape-character.dot
+
+###[ EMBEDDED HTML ]###########################################################
+
+  # https://graphviz.org/doc/info/lang.html#html-strings
```

Comment:
Optionally put in a ToDo for making this the Graphviz flavor of HTML, which IIRC is from the 2000s. Either here or at the actual `embed` location.

You don't need to do the work, but since you're already flagging future improvements, this could be added.